db/migrate/20220928150349_add_campaign_name_to_course.rb

```diff
@@ -1,5 +1,5 @@
 class AddCampaignNameToCourse < ActiveRecord::Migration[7.0]
   def change
-    add_column :course, :campaign_name, :integer
+    add_column :course, :campaign_name, :string, default: "no_campaign"
```

Comment:
Any rationale for this?

Reply:
Yep, the test specs would not seem to load the default from the initializer, played around and this seemed to work, also we should be able to set the default in the model, but that didn't work either...